requirements-freqai.txt

```diff
@@ -1,9 +1,14 @@
 # Include all requirements to run the bot.
--r requirements.txt
+-r requirements-hyperopt.txt
 
 # Required for freqai
 scikit-learn==1.1.2
 joblib==1.2.0
 catboost==1.1; platform_machine != 'aarch64'
 lightgbm==3.3.2
 xgboost==1.6.2
+torch==1.12.1
+stable-baselines3==1.6.0
+gym==0.21.0
+tensorboard==2.9.1
+sb3-contrib==1.6.0
```

Comment:
are we sure they all install on different systems (aarch64, armv7)? (that requires wheels to be available for all fo them)

Reply:
gym needs 0.21 to work with stable_baselines3, so we will keep it that way until stable_baselines3 updates their connection. 

I split out the install so that general FAI install wont automatically install all the RL dependencies.